### PR TITLE
xxxLightShadowFilter: format code + javadoc

### PR DESCRIPTION
**Pull request by capdevon**
* DirectionalLightShadowFilter: format code + javadoc
* PointLightShadowFilter: format code + javadoc
* SpotLightShadowFilter: format code + javadoc

Update License year
Removed broken links to unreachable sites

### CODE REVIEW

File: jme3-core/src/main/java/com/jme3/shadow/DirectionalLightShadowFilter.java [hunk 1]
```diff
@@ -57,32 +55,28 @@
...
-     * technique at <a
-     * href="http://http.developer.nvidia.com/GPUGems3/gpugems3_ch10.html">http://http.developer.nvidia.com/GPUGems3/gpugems3_ch10.html</a>
-     *
-     * @param assetManager the application's asset manager
-     * @param shadowMapSize the size of the rendered shadowmaps (512, 1024, 2048,
-     *     etcetera)
-     * @param nbSplits the number of shadow maps rendered (More shadow maps mean
-     *     better quality, fewer frames per second.)
+     * Creates a new instance of {@code DirectionalLightShadowFilter} with the
+     * specified parameters.
```
Comment (codex128) [inline jme3-core/src/main/java/com/jme3/shadow/DirectionalLightShadowFilter.java:68]: Just my opinion, but this statement seems a bit redundant.

File: jme3-core/src/main/java/com/jme3/shadow/DirectionalLightShadowFilter.java [hunk 2]
```diff
@@ -57,32 +55,28 @@
...
+     * @param shadowMapSize the size of the rendered shadow maps (512, 1024, 2048, etc...)
+     * @param nbSplits      the number of shadow maps rendered (more shadow maps = better quality, but slower)
      */
     public DirectionalLightShadowFilter(AssetManager assetManager, int shadowMapSize, int nbSplits) {
         super(assetManager, shadowMapSize, new DirectionalLightShadowRenderer(assetManager, shadowMapSize, nbSplits));
     }
 
     /**
-     * return the light used to cast shadows
+     * Return the light used to cast shadows.
```
Comment (codex128) [inline jme3-core/src/main/java/com/jme3/shadow/DirectionalLightShadowFilter.java:79]: Return -> Returns

File: jme3-core/src/main/java/com/jme3/shadow/DirectionalLightShadowFilter.java [hunk 3]
```diff
@@ -110,24 +104,22 @@ public float getLambda() {
...
     /**
-     * Adjusts the partition of the shadow extend into shadow maps.
-     * Lambda is usually between 0 and 1.
-     * A low value gives a more linear partition,
-     * resulting in consistent shadow quality over the extend,
-     * but near shadows could look very jagged.
-     * A high value gives a more logarithmic partition,
-     * resulting in high quality for near shadows,
-     * but quality decreases rapidly with distance.
+     * Adjusts the partition of the shadow extend into shadow maps. Lambda is
```
Comment (codex128) [inline jme3-core/src/main/java/com/jme3/shadow/DirectionalLightShadowFilter.java:107]: Do you think `<p>` tags should be added between the first and second sentence and the last sentence?

File: jme3-core/src/main/java/com/jme3/shadow/DirectionalLightShadowFilter.java (hunk 2 above)
Comment (capdevon) [reply jme3-core/src/main/java/com/jme3/shadow/DirectionalLightShadowFilter.java]: Done

File: jme3-core/src/main/java/com/jme3/shadow/DirectionalLightShadowFilter.java (hunk 1 above)
Comment (capdevon) [reply jme3-core/src/main/java/com/jme3/shadow/DirectionalLightShadowFilter.java:68]: The previous javadoc was very similar, I just changed it to the standard description of any constructor that you can find in any API, also suggested by Google Gemini.

**Review by codex128 — APPROVED** [review]

**Review by codex128 — APPROVED** [review]

**Pull request merged**